src/operator/subgraph/partition_graph.cc

```diff
@@ -46,13 +46,18 @@ using nnvm::NodePtr;
 using nnvm::NodeEntry;
 using nnvm::Graph;
 
+using NodeEntryMap =
```

Comment:
there is a templated NodeEntryMap @ 3rdparty/tvm/nnvm/include/nnvm/node.h:72 , you may be able to use it

Reply:
Good catch!